plugins/modules/k8s_exec.py

```diff
@@ -183,6 +183,7 @@ def execute_module(module, k8s_ansible_mixin):
     else:
         rc = int(err['details']['causes'][0]['message'])
 
+    module.deprecate("The 'return_code' return key is deprecated. Please use 'rc' instead.", version="4.0.0", collection_name="kubernetes.core")
```

Comment:
how to do manage redirection for this deprecation ? if someone is using ``community.kubernetes.k8s_exec`` for instance

Reply:
What do you mean? The deprecation warning will be printed the same if you are using community.kubernetes. Sure, it will say "kubernetes.core", but you will also have gotten a deprecation warning that community.kubernetes is deprecated. I don't think we want to try and do anything to have it print community.kubernetes because that would be a ton of work for a collection we no longer are supporting.

Reply:
agree.
I was just trying to understand.
Thanks